File: src/Data/Models/MediaFile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.Models
{
    public class MediaFile
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? Uri { get; set; }
        public string? ContentType { get; set; }
    }
}

File: src/LuminaPath.MobileApp/src/app/core/models/mediaFile.ts
export class MediaFile {
  id: number;
  name: string | null;
  uri: string | null;
  contentType: string | null;
  constructor() {
    this.id = 0;
    this.name = null;
    this.uri = null;
    this.contentType = null;
  }
}
